File: reto_6.py
#Escribe un programa que pida al usuario una serie de números (separados por comas) y luego muestre:
#El número mayor.
#El número menor.
a = input("ingresa 1 numero (1 de 5): ")
b = input("ingresa 1 numero (2 de 5): ")
c = input("ingresa 1 numero (3 de 5): ")
d = input("ingresa 1 numero (4 de 5): ")
e = input("ingresa 1 numero (5 de 5): ")
y = [int(a),int(b),int(c),int(d),int(e)] #parecia ocurrir un error si no se declaraban como int
print(f"el numero menor es {min(y)} y el numero mayor es {max(y)}")

# numeros = [int(num.strip()) for num in entrada.split(",")]

File: Retos_1_10/reto_6.js
//Escribe un programa que pida al usuario una serie de números (separados por comas) y luego muestre:
//El número mayor.
//El número menor.

const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,  
  output: process.stdout 
});
rl.question("Ingresa 5 números separados por comas: ", (input) => {
let array1 = input.split(',').map(Number);// el map se utiliza para reemplazar los elemntos con otro tipo de lemnteos, entest caso los convertimos en numeros ya uqe eran strings
console.log(`El numero menor es ${Math.min(...array1)} y el numero mayor es ${Math.max(...array1)}`)
//en Math.min(...array1) los 3 puntos separan el array, ya que la expresion espera numeros sueltos
rl.close();});